zerobuf/CMakeLists.txt

```diff
@@ -31,8 +31,13 @@ set(ZEROBUF_SOURCES
 list(APPEND CPPCHECK_EXTRA_ARGS
   --suppress=*:${CMAKE_CURRENT_LIST_DIR}/jsoncpp/*)
 
+if(MSVC)
+	#currently, there seems to be no equivalent to -Wno-shadow under MSVC
+	#--would have to test Intel compilers under WIN32
+else(MSVC)
```

Comment:
no need for repeating the 'MSVC' in else() and endif() (looks CMake < 2,6 too me :) )